src/bevent.c

```diff
@@ -570,15 +572,21 @@ int event_add_au_jb_stat(struct odict *od_parent, const struct call *call)
  * @param h   Event handler
  * @param arg Handler argument
  *
+ * @deprecated Use bevent_register()
+ *
  * @return 0 if success, otherwise errorcode
  */
 int uag_event_register(ua_event_h *h, void *arg)
 {
 	struct ua_eh *eh;
+	static int w = 3;
```

Comment:
Can you replace this magic number `3` with a enum constant, so it's more meaningful.